docs/source/models/gbd2017/causes/measles.rst

```diff
@@ -8,60 +8,105 @@ Disease Description
-transmitted by an infected person from 4 days prior to the onset of the rash
-to 4 days after the rash erupts [WHO]_, [CDC]_, [Wikipedia]_.
-
-Before the introduction of a measles vaccine in 1963 and widespread
-vaccination, major epidemics occurred approximately every 2–3 years, and
-measles caused an estimated 2.6 million deaths each year. Despite the
-availability of a vaccine, approximately 110,000 people died from measles in
-2017, mostly children under the age of 5 years. However, due to accelerated
-immunization activities, global measles deaths have decreased 80% during the
-period 2000–2017, from an estimated 545,000 to  110,000, and measles
-vaccination prevented an estimated  21.1 million deaths during
-2000–2017 [WHO]_.
+transmitted by an infected person from 4 days prior to the onset of the rash to
+4 days after the rash erupts [WHO]_, [CDC]_, [Wikipedia]_.
+
+Before the introduction of a measles vaccine in 1963 and widespread vaccination,
+major epidemics occurred approximately every 2–3 years, and measles caused an
+estimated 2.6 million deaths each year. Despite the availability of a vaccine,
+approximately 110,000 people died from measles in 2017, mostly children under
+the age of 5 years. However, due to accelerated immunization activities, global
+measles deaths have decreased 80% during the period 2000–2017, from an estimated
+545,000 to  110,000, and measles vaccination prevented an estimated  21.1
+million deaths during 2000–2017 [WHO]_.
 
 The `ICD 10`_ codes for measles are B05-B05.9, Z24.4, and ICD 9 codes are
 055-055.9, 484.0, V04.2, V73.2 [GBD-2017-YLD-Capstone-Appendix-1]_.
 
 .. _measles: https://en.wikipedia.org/wiki/Measles
 .. _ICD 10: https://en.wikipedia.org/wiki/ICD-10
 
+.. todo::
+
+   Add data about global vaccine coverage and efficacy. Perhaps start with these references:
+
+   - https://www.who.int/gho/mdg/child_mortality/situation_trends_measles_immunization/en/
+
+   - https://www.who.int/immunization/newsroom/measles-data-2019/en/
+
+
 Modeling Measles in GBD 2017
 ----------------------------
 
 .. todo::
 
    Add relevant detail about measles modeling process from
-   [GBD-2017-YLD-Capstone-Appendix-1]_ and from the CoD Appendix.
+   [GBD-2017-YLD-Capstone-Appendix-1]_ and from the `CoD Capstone
+   <http://dx.doi.org/10.1016/S0140-6736(18)32203-7>`_ Appendix. Note that each
+   country's vaccine coverage went into the estimation of measles incidence
+   rates, which are then multiplied by an average disease duration of 10 days to
+   compute prevalence.
+
+   Describe enough of the data sources and modeling process to verify that even
+   though  measles can lead to diarrhea or other causes that we include in our
+   Vivarium models, we won't be double counting mortality and morbidity from
+   these causes. For example, a death caused by diarrheal dehydration due to
+   measles should be counted in the GBD as a death due to measles, not as a
+   death due to diarrheal diseases.
```

Comment:
Check on the measles sequelae as well. I'm not sure what the non-fatal outcomes look like.  We've tended not to pay much attention to the non-fatal side of esp. under 5 diseases, but it sounds like some of the things here can persist well after someone recovers from measles and maybe that's important to think about.

Reply:
Sounds good, thanks for the suggestion.